crates/stackable-operator/src/logging/mod.rs

```diff
@@ -22,18 +25,31 @@ impl Default for TracingTarget {
 /// We force users to provide a variable name so it can be different per product.
 /// We encourage it to be the product name plus `_LOG`, e.g. `FOOBAR_OPERATOR_LOG`.
 /// If no environment variable is provided, the maximum log level is set to INFO.
+///
+/// Log output can be copied to a file by setting `{env}_DIRECTORY` (e.g. `FOOBAR_OPERATOR_DIRECTORY`)
+/// to a directory path. This file will be rotated regularly.
```

Comment:
> This file will be rotated regularly

Actually it looks like it defaults to no rotation: https://github.com/tokio-rs/tracing/blob/ba387ddc03f17eb91e70efe58ae007326ecb9cd4/tracing-appender/src/rolling/builder.rs#L41

Can we please explicitly set a rotation?

Reply:
Argh, you're right. It _used to be_ hourly, but I screwed up when migrating to the builder API instead.

Reply:
2c5e3548b0437bb9d6c9ff71c783c8a328fdd20c

Reply:
You "just" forget to push the commit ;)

Reply:
GODDAMNIT IT HAPPENED AGAIN

Reply:
2c5e3548b0437bb9d6c9ff71c783c8a328fdd20c